metrics_handler/main_test.py

```diff
@@ -0,0 +1,98 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from absl import logging
+from absl.testing import absltest
+import tensorflow as tf
+
+import alert_handler
+import main
+import metrics
+
+class CloudMetricsHandlerTest(absltest.TestCase):
+  def setUp(self):
+    self.logger = alert_handler.AlertHandler(
+      project_id=None,
+      write_to_logging=True,
+      write_to_error_reporting=False,
+      write_to_email=False)
+
+    self.temp_dir = self.create_tempdir().full_path
+    self.summary_writer = tf.summary.create_file_writer(self.temp_dir)
+
+    with self.summary_writer.as_default():
+      tf.summary.scalar("foo", 1, 0)
+      tf.summary.scalar("bar", tf.convert_to_tensor(1), 0)
+
+      tf.summary.scalar("foo", 2, 100)
+      tf.summary.scalar("bar", tf.convert_to_tensor(2), 100)
+
+    self.summary_writer.close()
+
+  def test_get_metrics_from_event_dir(self):
+    metrics_handler = main.CloudMetricsHandler(
+      test_name="test",
+      events_dir=self.temp_dir,
+      stackdriver_logs_link=None,
+      metric_collection_config={
+        'default_aggregation_strategies': ['final', 'min',]
+      },
+      regression_test_config={},
+      test_type=None,
+      accelerator=None,
+      framework_version=None,
+      logger=self.logger, # pass in absl logging module
```

Comment:
For this comment (and the matching one below):
1. Formatting:  2 spaces between the comma and the #, Capitalize first letter, end with period
2. Prefix with TODO if this is something we want to do. Right now we're passing AlertHandler, but the comment refers to absl logging, so I'm assuming this is a future intended change

Reply:
Oops, that's an old comment. Removed.